back/db/entrypoint/init.sql

```diff
@@ -11,3 +11,10 @@ CREATE TABLE IF NOT EXISTS news (
     created_date DATE NOT NULL,
     modified_date DATE NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS users (
+    uid         INT             AUTO_INCREMENT  PRIMARY KEY,
```

Comment:
I still think using `id` rather than `uid` is a better idea: `uid` means that user id, but the table's name is already `user`.

Reply:
ok